jobserv/api/worker.py

```diff
@@ -29,6 +31,12 @@ def _is_worker_authenticated(worker: Worker):
         parts = key.split(" ")
         if len(parts) == 2 and parts[0] == "Token":
             return worker.validate_api_key(parts[1])
+        if len(parts) == 2 and parts[0] == "Bearer":
+            try:
+                w = worker_from_jwt(parts[1])
+                return w.name == worker.name
+            except PyJWTError:
+                pass
```

Comment:
So, in a case of the auth failure (PyJWTError based exception or if `w.name != worker.name`) `worker_get` (HTTP GET workers/<name>/) returns 200, at least that's how I read `def worker_get(name):` implementation?

Reply:
good point. we should probably lock this down better. workers shouldn't be able to see each other (actually regular anonymous api users shouldn't either).